website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -1,165 +1,160 @@
+track how many auditable events you have lost.
+
+## Audit device limitations
+
+1. You cannot add filtering to an existing audit device.
+1. You can configure filtering when enabling one of the following supported audit device types:
+    - [file](/vault/docs/audit/file)
+    - [socket](/vault/docs/audit/socket)
+    - [syslog](/vault/docs/audit/syslog)
+1. You can only designate one auditing fallback device.
+
+## Filtering and test messages
+
+By default, Vault sends a test message to the audit device when you enable it.
+Depending on how you configure your filters, the default test message may fail
+the predicate expression and not write to the new device.
+
+You can determine whether the test message should appear in the sink for the 
+newly enabled audit device based on the following property
+table, which are common to all default test messages.
+
+Property      | Value
+------------- | ----------------
+`mount_point` | empty
+`mount_type`  | empty
+`namespace`   | empty
+`operation`   | `update`
+`path`        | `sys/audit/test`
+
+## `filter` properties for audit devices
+
+Filters can only reference the following properties of an audit entry:
+
+Property      | Example                             | Description
+------------- | ----------------------------------- | --------------------------
+`mount_point` | `mount_point == auth/oidc`          | Log all entries for the `auth/oidc` mount point
+`mount_type`  | `mount_type == \"kv-v2\"`           | Log all entries from `kv-v2` plugins
+`namespace`   | `namespace != admin/`               | Log all entries **not** in the admin namespace
+`operation`   | `operation == read`                 | Log all read operations
+`path`        | `path == auth/approle/login`        | Log all activity against the AppRole login path
+
+<Tip title="Root namespaces are unnamed">
+
+  Most namespace strings **must** end with a trailing slash to match correctly.
+  But the root namespace is technically unnamed and only matches to an empty
+  string. To match to the root namespace in your filter use `\"\"`. For example,
+  `namespace != \"\"` matches any audit entry **not** in the root namespace.
+
+</Tip>
+
+## A practical example
+
+Assume you already have an audit file called `vault-audit.log` but you want to
+filter your audit entries and persist all the key/value (`kv`) type events to a
+specific audit log file called `kv-audit.log`.
+
+To filter the events:
+
+1. Enable a `file` audit device with a `mount_type` filter:
+  ```shell-session
+  vault audit enable                \
+    -path kv-only                   \
+    file                            \
+    file_path=/logs/kv-audit.log    \
+    filter="mount_type == kv"
+  ```
+1. Enable a fallback device:
+  ```shell-session
+  vault audit enable                \
+    fallback=true                   \
+    -path=my-fallback               \
+    -description="fallback device"  \
+    file file_path=/tmp/kv-audit.fallback.log 
+  ```
```

Comment:
`fallback` has to come at the end where the kv config options go.

```suggestion
  ```shell-session
  vault audit enable                \
    -path=my-fallback               \
    -description="fallback device"  \
    file                            \
    fallback=true                   \
    file_path=/tmp/kv-audit.fallback.log
  ```
```